src/utils/image_build.py

```diff
@@ -51,10 +47,9 @@ def build_and_upload_images(
         print("Directory src/templates does not exist.")
         print("Please run this script from the project root.")
         return False
-    os.chdir("src/templates")
 
     # Setup buildkit
-    builder_name = "warnet-builder"
+    builder_name = "bitcoind-builder"
```

Comment:
in "fixup! images: unified alpine-based dockerfile" (https://github.com/bitcoin-dev-project/warnet/pull/261/commits/6070ea73f07dbccd0e08a1baec44ebe974f94e89):

in the previous commit this was called `warnet-bitcoind-builder`, not sure if that makes a difference. Also noticed you're removing the script in this commit, might be worth just dropping the script entirely from the PR?

Reply:
Which script do you mean? This PR attempts to clean a load of them out already?

Reply:
I could have sworn you were adding `build-bitcoind.sh` in an earlier commit and then removing it in this one.. but I don't see it now? Either I'm losing my marbles or you already removed it haha